x-pack/test/plugin_functional/test_suites/resolver/index.ts

```diff
@@ -4,24 +4,174 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import expect from '@kbn/expect';
+import { WebElementWrapper } from '../../../../../test/functional/services/lib/web_element_wrapper';
+
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getPageObjects, getService }: FtrProviderContext) {
+const expectedDifference = 0.09;
+
+export default function ({
+  getPageObjects,
+  getService,
+  updateBaselines,
+}: FtrProviderContext & { updateBaselines: boolean }) {
   const pageObjects = getPageObjects(['common']);
   const testSubjects = getService('testSubjects');
+  const screenshot = getService('screenshots');
+  const find = getService('find');
+  const browser = getService('browser');
 
   describe('Resolver test app', function () {
     this.tags('ciGroup7');
 
-    beforeEach(async function () {
+    // Note: these tests are intended to run on the same page in serial.
+    before(async function () {
       await pageObjects.common.navigateToApp('resolverTest');
+      // make the window big enough that all nodes are fully in view (for screenshots)
+      await browser.setWindowSize(1920, 1200);
```

Comment:
Thoughts on why expectedDifference would change based on a monitor being connected when you explicitly set the size like this? 🤔

Reply:
yah, the pixel density changes when a monitor is plugged in. IIRC, It'll be around 220 PPI on my built in display and 110 PPI on my 4K 34" ultra wide displayport 2 monitor.

Reply:
Makes sense, kind of annoying though 😂